src/field/input_features.md

```diff
@@ -6,100 +6,114 @@
+
 Surveyed features are added to the active layer, which is shown just above the bottom panel. You can change the active layer by tapping on it and selecting one from the list of editable layers in the project.
-![active layer](./input-active-layer.jpg "Active layer")
+![active survey layer](./input-active-layer.jpg "Active survey layer")
 
 ### Capture points
-The crosshairs you will see on your map will be used as the recorded location. You can change the position of your point by pinching and dragging the background map. If you want to recenter the map to your current position, tap the **GPS** button. 
-
-To capture your point, press **Add Point**. A form will open where you can fill in the attributes.
+To capture a new point feature, tap **Add** on the bottom panel. A form will open where you can fill in the attributes.
 
-![Capturing point](../tutorials/capturing-first-data/merginmaps-mobile-default-point-position.jpg)
+![Add point in Mergin Maps Input](./input-add-point.jpg "Add point in Mergin Maps Input")
 
 ### Capture lines or areas
-There are two methods of capturing lines and areas:
-  - Using **Add point** to capture vertices of your line or area:
-    ![Capturing lines](./input-capture-line.png)
-  
-  - Tracking your position using **GPS**. Long press the **GPS** button to turn the tracking on and the vertices will be captured automatically based on your position. You can change how often you want to capture a vertex in **Line rec. interval** in [**Settings**](./input_ui/#recording-settings). 
-    
-    To turn the tracking off, long press the **GPS** button again.
-    ![GPS tracking](./input-gps-tracking.png)
-
-Once you are finished, press **Done**. If your feature is an area, the shape will be automatically closed by connecting the last and the first vertex. 
-
-### Snapping features
-<Badge text="since Input 1.6.0" type="tip"/>
+There are two methods of capturing lines and areas: adding vertices one by one or using the *streaming mode* to capture features based on your position.
 
-Snapping can be enabled in your <MainPlatformName /> project to make the field survey easier. [How to Set Up Snapping](../gis/snapping/) will guide you through the snapping options.
+#### Adding points to survey vertices of lines or areas
+Tap **Add** to capture vertices of your line or area. If you want to change the position of the last vertex, tap **Remove**, move the vertex to the correct place and  **Release** it. **Undo** can be used to revert last changes.
+
+Once the survey of the feature is completed, tap **Done** and fill in the form.
+![Capturing lines](./input-capture-line.jpg)
+
+### Streaming mode to survey lines or areas
+Lines and areas can be also captured automatically based on your position. To turn on the streaming mode, long press the **GPS** button. To turn it off, long press the **GPS** button again.
```

Comment:
A `::: tip/note` might be beneficial here saying that one needs to be in the recording mode. If you hold the GPS button in view mode, nothing happens.

Reply:
I understand that it is mentioned earlier, it is just that people might google/search for streaming mode and end up here without reading the previous text